lib/discover/client_test.go

```diff
@@ -1,239 +0,0 @@
-// Copyright (C) 2014 The Syncthing Authors.
```

Comment:
Every time we delete a file with _test.go suffix, a kitten dies.
But I guess it's all for a greater cause.